### PR TITLE
AP: Announce will now work

### PR DESCRIPTION
**Pull request by annando**
Announces on AP are different from Friendica or Diaspora. So we will have to do some UI changes in the future.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -46,7 +46,7 @@
 	System::httpExit(503, ['title' => 'Error 503 - Service Temporarily Unavailable', 'description' => 'System is currently overloaded. Please try again later.']);
 }
 
-if (strstr($a->query_string, '.well-known/host-meta') and ($a->query_string != '.well-known/host-meta')) {
+if (strstr($a->query_string, '.well-known/host-meta') && ($a->query_string != '.well-known/host-meta')) {
```
Comment (MrPetovan) [inline index.php]: 😄

**Comment by MrPetovan** [comment]
Follow-up to #5794

**Comment by annando** [comment]
"Announce" is the AP terminology for reshared items. On Diaspora and Friendica they are a new post with a new URI (or GUID), on AP we only transmit this "Announce" message, but we don't create a new post.

**Comment by tobiasd** [comment]
Hopefully we keep the quote re-share.

**Comment by annando** [comment]
@MrPetovan With this change we now send an "Announce" into the AP world, while we keep on sending reshares to Friendica and Diaspora. But in the future we have to change something. But since I'm not very good in UX (or is it UI?), I currently don't have a clue how to handle it.

**Comment by MrPetovan** [comment]
So we are loosing quote re-shares with ActivityPub. Can we just have the shared post link in a top-level post instead for those?

**Comment by annando** [comment]
When I did it correctly, then quote reshares will currently work as well. They shouldn't be translated to announces.

**Comment by MrPetovan** [comment]
When or If?

**Comment by annando** [comment]
Good question, I always mix these two words :-) Let's say it this way: It **should** work.

**Pull request merged**